topography/topography.F90

```diff
@@ -97,11 +101,12 @@ module topography_mod
                          water_file = 'DATA/navy_pctwater.data'
    namelist /topography_nml/ topog_file, water_file
 ! </NAMELIST>
-
+#ifndef use_mpp_io
    integer, parameter    :: TOPOG_INDEX = 1
    integer, parameter    :: WATER_INDEX = 2
```

Comment:
Put the `#ifndef` below this.  These are namelist variables, so they need to be used regardless of IO